src/main/java/backend/Task/DeadlineTask.java

```diff
@@ -0,0 +1,33 @@
+package backend.Task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a task with a deadline. A <code>DeadlineTask<code> object has a
```

Comment:
I think the `<code>` at the end should be `</code>`.
This was found in the other code files as well. 

```suggestion
 * Represents a task with a deadline. A <code>DeadlineTask</code> object has a
```